src/serialization/sb2_specmap.js

```diff
@@ -1252,7 +1252,7 @@ const specMap = {
         ]
     },
     'contentsOfList:': {
-        opcode: 'data_list',
+        opcode: 'data_listcontents',
```

Comment:
Curious about this change, was this broken before?

Reply:
Yup, it was.